runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -364,21 +365,24 @@
      */
     private static final String mp = "corfu.runtime.";
     @Getter
-    private static final String mpASV = mp + "as-view.";
+    private static final String mpAsv = mp + "as-view.";
     @Getter
-    private static final String mpLUC = mp + "log-unit-client.";
+    private static final String mpLuc = mp + "log-unit-client.";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)